docs/sysdef_examples.rst

```diff
@@ -0,0 +1,128 @@
+.. _sysdef_examples:
+
+==========================
+System Definition Examples
+==========================
+
+.. contents::
+   :depth: 4
+
+Basic Examples
+==============
+
+Creating a basic system definition file
+---------------------------------------
+A system definition can be created with either an initial target of either Windows or Linux.
```

Comment:
"A system definition can be created with a target operating system of Windows or Linux_x64."